front/src/api/vehicles.ts

```diff
@@ -219,7 +219,7 @@ function randomVehicle(): IVehicleResponse {
     };
   }
   if (Math.random() > 0.2) {
-    const provider = Math.random() > 0.5 ? "JUMP" : "Metro Bike";
+    const provider = Math.random() > 0.5 ? "JUMP" : "Metro Bicycle";
```

Comment:
Really?

Reply:
Why not, consistency until the end.

Reply:
Metro Bike is a company actually operating in LA https://bikeshare.metro.net/

Reply:
You have seen nothing.